orbit-2-core/src/main/java/com/babylon/orbit2/Context.kt

```diff
@@ -23,4 +23,18 @@ package com.babylon.orbit2
  * @property event The current event being processed
```

Comment:
Shall we change the wording here?

> @property state The state captured at the point when the operator is executed

No longer "current" per se

Reply:
hahaha, yeah